### ISSUE TITLE
Please load the latest phase by default

### ISSUE THREAD

**Issue by GijsTimmers** [zond/android-diplicity#61]
It looks like diplicity doesn't load the latest phase in all cases. This can be confusing.

Video: https://streamable.com/hj5ax

**Comment by tttppp** [comment]
Yes, I've noticed this too. I think it's possibly related to the back stack problem. I think it might happen if you've recently had the previous phase open.

Thanks for the screen recording of this!

**Comment by GijsTimmers** [comment]
perhaps some of the refresh code should be moved from `OnCreate()` / `OnRestart()` to `OnResume()`

reference: https://developer.android.com/reference/android/app/Activity#ActivityLifecycle

**Comment by zond** [comment]
Feel free to try it out and see if it works, if you like it please describe
what kind of changes it creates, and we can discuss if we should release it?

On Jan 6, 2019 21:15, "GijsTimmers" <notifications@github.com> wrote:

perhaps some of the refresh code should be moved from OnCreate() /
OnRestart() to OnResume()

reference:
https://developer.android.com/reference/android/app/Activity#ActivityLifecycle

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/zond/android-diplicity/issues/61#issuecomment-451771012>,
or mute the thread
